src/formatDate.js

```diff
@@ -50,7 +50,29 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateToArr = date.split(fromFormat[3]);
```

Comment:
The variable's name should say **what** is stored in it, and not **"how"** or **" where"**.
For example `arrOfDate`; 😄